README.md

```diff
@@ -1,2 +1,221 @@
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+---
+
+<!-- PROJECT DESCRIPTION -->
+
+#  Rent For a Day <a name="about-project"></a>
+
+**"Rent For a Day"** is a reservation-based application to rent a property/accommodation for a day. This project is a React/Redux Toolkit application, and it allows users to register and log in with their credentials, see all the items registered, and add or delete items to the inventory.
+
+Two repositories were created for its construction:
+- [A backend repository](https://github.com/Zven94/RentForADay-app) with an API built with the [Ruby on Rails](https://rubyonrails.org/) Framework with a [PostgreSQL](https://www.postgresql.org/) database.
+- [A frontend repository](https://github.com/Zven94/rentforaday-front-end) with a React/Redux Toolkit application from which the created API is consumed.
+
+Additionally, the API documentation is available [here](https://rent-for-a-day.onrender.com/api-docs/index.html).
+
+---
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack
+
+The project is built using the following technologies:
+
+* Ruby on Rails.
+* PostgreSQL.
+* RSpec.
+* Rubocop.
+* Devise.
+
+<br>
+
+---
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [x] User Registration: Register an account securely.
+- [x] Item Management: The user can add and delete items to the inventory.
+- [x] User-Friendly Interface: Navigate with ease.
+- [x] Responsive Design: Accessible on various devices.
+
+
+<ul>
+  <li>Ruby on Rails</li>
+  <li>Linters
+    <ul>
+      <li>Rubocop</li>
+    </ul>
+  </li>
+  <li>Follow Microverse's list of best practices</li>
+</ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ---
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- 🎤 [Project presentation: video demo](https://www.loom.com/share/a57b45a0708545d9bac18071ead813e2?sid=bbd6e390-585b-487b-8157-76dc5cb071ee)
+- [Live Demo Link](https://budget-app-st58.onrender.com)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need to have:
+
+- Installed Ruby and Ruby on Rails.
+- If you're using Windows, have installed WSL.
+- If you're using MacOS, an option is using UTM.
+
+### Setup

```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍